docs/standard/serialization/binaryformatter-migration-guide/winforms-migration-guide.md

```diff
@@ -0,0 +1,89 @@
+
+### Drag-and-drop feature
+
+If your drag-and-drop scenario involves types that aren't intrinsically handled during serialization and deserialization, `BinaryFormatter` is used when [`Control.DoDragDrop`](/dotnet/api/system.windows.forms.control.dodragdrop) is called, and when [`DataObject.GetData`](/dotnet/api/system.windows.dataobject.getdata) is called to retrieve a type that's been dragged out of process. With the `BinaryFormatter` removal, you'll now see a string about `BinaryFormatter` being removed when you drop a dragged item in another process for drag-and-drop operations with types that aren't intrinsically handled.
+
+### The Windows Forms Designer
+
+The Windows Forms Designer also uses `BinaryFormatter` internally for ResX serialization and deserialization, and for CodeDom.
+
+Types and properties might participate in serialization without you realizing due to the standard behavior of the Windows Forms Designer. One way that `BinaryFormatter` is used that you might not be aware of is when a property on a <xref:System.Windows.Forms.UserControl> is introduced and that control is already populated at design time. It's likely that that data gets serialized into resource files. Consider the following conditions:
+
+- A property contains data at the time when a Form in the Designer is saved.
+- That property is not attributed with `DesignerSerializationVisibility(false)`.
+- That property is not read-only.
+- That property does not have a DefaultValueAttribute.
+- That property does not have a respective `bool ShouldSerialize[PropertyName]` method that returns `false` at the time of the CodeDOM serialization process. (Note: the method can have `private` scope.)
+
+If these statements are true, the Designer determines if that property’s type has a type converter. If it does, the Designer uses the type converter to serialize the property content. Otherwise, it uses `BinaryFormatter` to serialize the content into the resource file.
+Windows Forms has added analyzers along with code fixes to help bring awareness to this type of behavior where `BinaryFormatter` serialization might be occurring without the developer’s knowledge.
```

Comment:
Does it make sense to add a screen shot with such property detected by the analyzer?

Reply:
I don't think it is necessary given the conditions outlined seem clear when this analyzer would show, but I can do this addition